backend/src/main/java/heartbeat/client/dto/pipeline/buildkite/DeployInfo.java

```diff
@@ -21,6 +21,8 @@ public class DeployInfo {
 
 	private String state;
 
+	private Boolean isPipelineCanceled;
```

Comment:
maybe boolean is enough ? Or question: what is the benefit using Boolean here

Reply:
yes,boolean is enough, already change it